admin-ui/src/components/token-dialog.tsx

```
@@ -84,7 +102,7 @@ export function TokenDialog(): React.JSX.Element {
           <DialogTitle>{t("tokenDialog.title")}</DialogTitle>
           <DialogDescription>
             <Trans i18nKey="tokenDialog.description">
-              Used as <code>x-admin-token</code> when calling <code>/api/admin/*</code>. Stored in <code>sessionStorage</code>.
+              Used as <code>x-admin-token</code> when calling <code>/api/admin/*</code>. Stored in <code>localStorage</code>.
```

ESLint

[error] Insert `{"·"}⏎·············`

(prettier/prettier)